src/server/sitemap/useCases/générerSitemap.useCase.test.ts

```diff
@@ -4,17 +4,352 @@ import { aFicheMetierNomMetierList, anArticlePathList, aSitemap } from '~/server
 import { GénérerSitemapUseCase } from '~/server/sitemap/useCases/générerSitemap.useCase';
 
 describe('GénérerSitemapUseCase', () => {
-	it('génère le xml contenant le sitemap',  async() => {
-		const cmsRepository = aStrapiCmsRepository();
-		cmsRepository.listAllArticleSlug = jest.fn().mockResolvedValue(createSuccess(anArticlePathList()));
-		cmsRepository.listAllFicheMetierNomMetier = jest.fn().mockResolvedValue(createSuccess(aFicheMetierNomMetierList()));
-		const générerSitemapUseCase = new GénérerSitemapUseCase(cmsRepository);
-		const baseUrl = 'http://localhost:3000';
+	describe('quand le feature flip de Formation et Apprentissage n‘est pas actif', () => {
+		beforeEach(() => {
+			process.env = {
+				NEXT_PUBLIC_FORMATION_LBA_FEATURE: '0',
+			};
+		});
+		it('génère le xml contenant le sitemap',  async() => {
+			const cmsRepository = aStrapiCmsRepository();
+			cmsRepository.listAllArticleSlug = jest.fn().mockResolvedValue(createSuccess(anArticlePathList()));
+			cmsRepository.listAllFicheMetierNomMetier = jest.fn().mockResolvedValue(createSuccess(aFicheMetierNomMetierList()));
+			const générerSitemapUseCase = new GénérerSitemapUseCase(cmsRepository);
+			const baseUrl = 'http://localhost:3000';
 
-		const expected = aSitemap();
+			const expected = aSitemap();
 
-		const result = await générerSitemapUseCase.handle(baseUrl);
+			const result = await générerSitemapUseCase.handle(baseUrl);
 
-		expect(result).toEqual(expected);
+			expect(result).toEqual(expected);
```

Comment:
proposition:
`expect(result).not.toContain('<loc>http://localhost:3000/formations/apprentissage</loc>')`